client/public/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "short_name": "React App",
+  "short_name": "FightPandemics",
   "name": "Create React App Sample",
```

Comment:
Let's say "FightPandemics" here too

Reply:
https://github.com/FightPandemics/FightPandemics/issues/442#issuecomment-632919293

Reply:
Do you think we should merge for now? Or should we wait for more inputs?

Reply:
will check if there's any more suggestions

Reply:
updated